App/automotive/src/test/java/com/amos/infotaimos/model/TimerItemTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.amos.infotaimos.model
```

Comment:
Sollte Testspezifischer Code nicht auch in das Test Package?

Reply:
Test package?
Wandern die Tests nicht ins gleiche Package wie die zu testende Klasse, zumindest hab ich das nur so gesehen bisher ^^

Reply:
Bei den bisherigen Tests haben wir auch immer das Package von der zu testenden Klasse genommen